src/test/kotlin/coverosR3z/timerecording/EnteringTimeBDD.kt

```diff
@@ -19,7 +20,12 @@ import org.junit.Test
  */
 class EnteringTimeBDD {
 
-    val cua = CurrentUserAccessor()
+    val currentUserAccessor = CurrentUserAccessor()
+
+    @Before
```

Comment:
I don't know if this is practical but sharing this before between BDDs would be interesting, since it's repeated

Reply:
Sure, it's all good.

Reply:
But how

Reply:
Well, admittedly it's not going to be a huge gain.  The Before tag goes on a function.